include/rovio/PoseUpdate.hpp

```diff
@@ -60,8 +60,10 @@ class PoseUpdateMeas: public LWF::State<LWF::VectorElement<3>,LWF::QuaternionEle
  public:
   static constexpr unsigned int _pos = 0;
   static constexpr unsigned int _att = _pos+1;
+  M3D pos_cov_; // Will be used to scale the update covariance according to the measurement
```

Comment:
I agree that off-diagonal blocks become an issue if either position or attitude is disabled. But those blocks can easily be set to zero in such a case. Do I miss something?